.github/workflows/android-build.yml

```diff
@@ -65,6 +70,7 @@ jobs:
       - name: Create Build
         run: |
           cd ${{ inputs.ANDROID_SERVICE_LOCATION }}
+          cd ${{ inputs.BUILD_SCRIPT }}
           ./android-build.sh
```

Comment:
its not a directory it is a script.

please cd and remove .sh file and  ./${{ inputs.BUILD_SCRIPT}}.sh